src/transformers/pipelines/__init__.py

```diff
@@ -302,7 +302,7 @@
     },
     "zero-shot-audio-classification": {
         "impl": ZeroShotAudioClassificationPipeline,
-        "tf": (TFAutoModel,) if is_tf_available() else (),
+        "tf": (),
```

Comment:
To avoid the CI failure
```bash
tests/pipelines/test_pipelines_common.py::PipelineUtilsTest::test_load_default_pipelines_tf
(line 559)  KeyError: 'tf'
```
(at least for now)